proposals/lip-0067.md

```diff
@@ -210,13 +209,11 @@ Secret recovery phrase: `target cancel solution recipe vague faint bomb convince
 ```json
 {
   "crypto": {
-    "version": "1",
     "ciphertext": "866c6f1cab3ef67514bdc54cf0143b8b824ebe7c045efb97707c158c81d313cd1a6399b7aa3002248984d39ea2604b0263fe7bdbd8cb04286a9cbd2d353fc79908daab9af04b2528bf4f06a82d79483c",
```

Comment:
If I understand well, ciphertext is calculated using the cipher function, so since it changes, this has to change now I guess..

Btw, I realize it is never explicitly stated in the LIP, how "ciphertext" is calculated, perhaps we should add a sentence on it.

Reply:
Line 85 gives the description. Do you want to add more explicit description in "ciphertext" section also?

Reply:
Yep, that would be good. Like one more sentence. 

Also, for the flow of the text, might be better to explain first the explanation of `ciphertext` and then of `cipher` and `cipherparams`. Because somehow we now first discuss how to decrypt and then what is the excrypted message (which should be decrypted..).